src/Spryker/Zed/ProductStorage/Persistence/ProductStorageQueryContainerInterface.php

```diff
@@ -19,12 +19,26 @@ interface ProductStorageQueryContainerInterface extends QueryContainerInterface
      *
      * @api
      *
+     * @deprecated Use {@link \Spryker\Zed\ProductStorage\Persistence\ProductStorageQueryContainerInterface::queryProductAbstractsByIds()} for better performance.
+     *
      * @param array<int> $productAbstractIds
      *
      * @return \Orm\Zed\Product\Persistence\SpyProductAbstractLocalizedAttributesQuery
      */
     public function queryProductAbstractByIds(array $productAbstractIds);
 
+    /**
+     * Specification:
+     * - TODO: Add method specification.
+     *
+     * @api
+     *
+     * @param array<int> $productAbstractIds
+     *
+     * @return \Orm\Zed\Product\Persistence\SpyProductAbstractLocalizedAttributesQuery
+     */
+    public function queryProductAbstractsByIds(array $productAbstractIds);
```

Comment:
Could you also add a return type if we introduce a new method?

Reply:
Sure. I did not add it, because it was also not present in the existing method.

Reply:
done